blog/2024-10-21-october-releases.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: "October 2024 Releases"
+slug: "2024-10-21-october-releases"
+authors: [dashagurova]
+tags: [announcement, conda-package-handling, conda-package-streaming]
+description: |
+  conda-package-handling 2.4.0 and conda-package-streaming 0.11.0 have been released! 🎉
+image: img/blog/2024-10-21-october-releases/banner.png
+---
+
+The October 2024 releases included updates to conda-package-handling and conda-package-streaming! 🎉 All of these have been released to both `main` and `conda-forge` channels.
+
+## Changes in conda-package-handling [2.4.0](https://github.com/conda/conda-package-handling/releases/tag/2.4.0)
+
+To update conda-package-handling to the latest version, run:
+
+```bash
+conda install -n base conda-package-handling=2.4.0
+```
+<!-- truncate -->
+
+### ✨ What's New? ✨
+
+- Expose API and CLI for which components will be listed as part of `cph list`.
```

Comment:
I'm not entirely sure what this means. Like, the components of the API and CLI are now able to be listed by that command? Or something else?

Reply:
@dholth can you help clarify this for me?

Reply:
reviewed original PR and rephrased as:

> Expose API keyword and CLI argument to control which component(s) of the `.conda` artifact is listed in `cph list`.